qcodes/instrument_drivers/Lakeshore/Lakeshore_model_335.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict
+
+import pyvisa as visa
```

Comment:
To not depend on implementation details of imports in the pyvisa module you should also explicitly import pyvisa.constants

I would prefer to get rid of the pyvisa as visa alias and just import pyvisa as is